packages/zoe/test/unitTests/contracts/test-callSpread.js

```diff
@@ -32,25 +39,40 @@ function makeFakePriceAuthority(
     return freshestPrice;
   }
 
+  function getRecentPrice(timer, desiredPriceBrand, underlyingAmount) {
+    const underlyingValue = underlyingAmountMath.getValue(underlyingAmount);
+    return E(timer)
+      .getCurrentTimestamp()
+      .then(now => {
+        const price = priceFromSchedule(now);
+        const strikePrice = strikeAmountMath.make(price * underlyingValue);
+        return quoteMint.mintPayment(
+          quote.make(
+            harden([
+              {
+                Asset: underlyingAmount,
+                Price: strikePrice,
+                timer,
+                timestamp: now,
+              },
+            ]),
+          ),
+        );
+      });
+  }
+
```

Comment:
Why define `getRecentPrice` when you only need `priceAtTime` to test?

Reply:
I simplified the helper function.